akka-persistence-typed/src/main/scala/akka/persistence/typed/scaladsl/Effect.scala

```diff
@@ -86,7 +85,7 @@ object Effect {
    * @see [[Effect.thenUnstashAll]]
    */
   def unstashAll[Event, State](): Effect[Event, State] =
-    none.andThen(SideEffect.unstashAll[State]())
+    CompositeEffect(none.asInstanceOf[Effect[Event, State]], SideEffect.unstashAll[State]())
```

Comment:
`none[Event, State]` ?

Reply:
I got variance problems when I did that. I think that is the reason for the explicit casts above as well.